pkg/config/config.go

```diff
@@ -1075,6 +1074,8 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("runtime_security_config.activity_dump.local_storage.compression", true)
 	config.BindEnvAndSetDefault("runtime_security_config.activity_dump.remote_storage.formats", []string{"json"})
 	config.BindEnvAndSetDefault("runtime_security_config.activity_dump.remote_storage.compression", true)
+	config.BindEnvAndSetDefault("runtime_security_config.activity_dump.syscall_monitor.enabled", true)
```

Comment:
Is it expected to be enabled by default?

Reply:
For now yes, we're testing it in multiple environment and hope to be able to enable it by default on merge 👍🏻